tests/e2e/tests/pilot/testdata/v1alpha3/rule-503test-update-virtualservice.yaml

```diff
@@ -13,11 +13,11 @@ spec:
           host: c
           subset: v3
           port:
-            name: http
+            number: 80
```

Comment:
Do we need some test with name as well ? 

Also, in ServiceEntry - how will it work ? The endpoint port is using the service ports as keys to be mapped to the real port.

Reply:
names going away in route.Destination.. should have zero impact on service entry